### PR TITLE
update links

### PR DESCRIPTION
**Pull request by evanlivingston**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,33 @@ Here you can find examples that demonstrate how to use PolySync's APIs.
...
 - [Serial Reader](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/SerialReader) - Read data from a serial device. 
 - [Serial Writer](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/SerialWriter) - Write data to a serial device.
+- [CAN Reader](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/CANReader) - Read data from a CAN device. 
+- [CAN Writer](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/CANWriter) - Write data to a CAN device.
+- [Video Device Viewer](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/VideoDeviceViewer) - Interact with a Video Device.
+- [Video Device With Encode/Decode](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/VideoEncodeDecode) - Communicate with a video device, encode, and decode video data.
+
+## Utilities and Cool Stuff
+
+- [Data Generator](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/DataGenerator) - Publish various provided sensor data message.
```
Comment (ltbj) [inline README.md]: Description should be more along the lines of "Publish various PolySync message types with pseudo-random data"

File: README.md [hunk 2]
```diff
@@ -7,14 +7,33 @@ Here you can find examples that demonstrate how to use PolySync's APIs.
...
 - [HelloWorldPublisher](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/HelloWorldPublisher) - Publish a message from a node to the PolySync bus.
 - [HelloWorldSubscriber](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/HelloWorldSubscriber) - Subscribe a node to messages on the PolySync bus.
-- [Coordinate Transformation](www.google.com) - Use the Transform API to transform coordinate frames using a stack.
+- [Publish & Subscribe](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/PublishSubscribe) - Use publish/subscribe routines.
+- [Coordinate Transformation (single)](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/SingleTransform) - Use the Transform API to transform coordinate frames.
+- [Coordinate Transformation (stack)](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/TransformStack) - Use the Transform API to transform coordinate frames using a stack.
+- [Sample Application](https://github.com/PolySync/PolySync-CPP-Examples/tree/master/SampleApplication) - Creeate an application node.
```
Comment (ltbj) [inline README.md:13]: Create*

**Comment by CrockAgile** [comment]
Approved! Looks nice.

File: DataGenerator/DataGenerator.cpp [hunk 3]
```diff
@@ -24,6 +24,12 @@
  */
 
 /**
+ * \example DataGenerator.cpp
+ *
+ * PolySync CAN API Reader Example.
```
Comment (ltbj) [inline DataGenerator/DataGenerator.cpp:29]: Why are we referencing the CAN API here?

**Pull request merged**